src/main/java/dev/mlml/Utils.java

```diff
@@ -12,4 +12,44 @@ public static boolean stringIsUserMention(String input) {
     public static boolean stringIsChannelMention(String input) {
         return input.matches("^<#\\d{17,19}>$");
     }
+
+    // This method takes in time in milliseconds and returns a formatted string in H:M:S
```

Comment:
If you're going to add comments, you should use JavaDocs

Reply:
Bro can't let me catch a break. Do you want me to remove it or nah, I am not learning JavaDocs now?